### PR TITLE
[Slider] Add examples to BUILD file

### LINKED ISSUE #6239: Internal issue: b/122530390
This was filed as an internal issue. If you are a Googler, please visit [b/122530390](http://b/122530390) for more details.

---
#### Internal data
- Associated internal bug: [b/122530390](http://b/122530390)

### PR DESCRIPTION
**Pull request by codeman7**
Closes #6239

### CODE REVIEW

File: components/BottomAppBar/BUILD [hunk 1]
```diff
@@ -15,6 +15,8 @@
 
 load(
     "//:material_components_ios.bzl",
+    "mdc_examples_objc_library",
+    "mdc_examples_swift_library",
     "mdc_extension_objc_library",
     "mdc_objc_library",
```
Comment (romoore) [inline components/BottomAppBar/BUILD]: Please split this to another PR.

Comment (codeman7) [reply components/BottomAppBar/BUILD]: Done

**Review by romoore — APPROVED** [review]
